### PR TITLE
fix/prevent nan  when rendering rowan stats

### PR DESCRIPTION
**Pull request by alanrsoares**

### CODE REVIEW

**Review by oieduardorabelo — APPROVED** [review]
Screen Shot 2022-07-19 at 9 52 40 AM

File: apps/dex/src/layouts/Nav/Nav.tsx [hunk 1]
```diff
@@ -151,4 +118,48 @@ const Header = () => {
...
+          <span className="ml-3">/</span>
+        </>
+      ),
+    },
+    {
+      id: "tvl",
+      icon: <LockIcon />,
+      label: (
+        <>
+          {isLoadingTVL || isNaN(TVL) ? "..." : formatNumberAsCurrency(TVL)} TVL
```
Comment (oieduardorabelo) [inline apps/dex/src/layouts/Nav/Nav.tsx:144]: i first i tried to find the `isNaN` function/pkg... then I remembered we have `Number.isNaN` and `.isNaN` is a lias...classic JS

Comment (alanrsoares) [reply apps/dex/src/layouts/Nav/Nav.tsx:144]: isNaN has been a global since I first used js

Comment (oieduardorabelo) [reply apps/dex/src/layouts/Nav/Nav.tsx:144]: sure thing, I highlighted that it can be confusing, behind the scenes is an alias to `Number.isNaN` 😆

**Pull request merged**